internal/fuse/file.go

```diff
@@ -84,30 +91,25 @@ func (f *file) Attr(ctx context.Context, a *fuse.Attr) error {
 
 func (f *file) getBlobAt(ctx context.Context, i int) (blob []byte, err error) {
 	debug.Log("getBlobAt(%v, %v)", f.node.Name, i)
-	if f.blobs[i] != nil {
-		return f.blobs[i], nil
-	}
-
-	// release earlier blobs
```

Comment:
This loop has the interesting property that it doesn't release blobs when accessing files backwards. The main benefit here is that if a program keeps accessing files chunks before and after a blob boundary, then after one or two accesses both used blobs are kept in the cache.

It's not really optimal and I think a simple LRU cache could do a lot better.

Reply:
That's a good point. I'll install an LRU cache.

Reply:
Actually, when the same part of a file is accessed multiple times, shouldn't the kernel cache that?